client/src/js/services/VoucherService.js

```diff
@@ -30,6 +31,13 @@ function VoucherService(PrototypeApiService) {
   }
 
   /**
+   * Wraps the prototype report method.
+   */
+  function report(id, filetype) {
```

Comment:
This will be hooked up automatically :).  I don't think you actually need to write this method.  If you delete it, does the modal still work?